src/cryptography/hazmat/asn1/asn1.py

```diff
@@ -49,13 +55,30 @@ def _is_union(field_type: type) -> bool:
         if hasattr(types, "UnionType")
         else (typing.Union,)
     )
-    return typing.get_origin(field_type) in union_types
+    return get_type_origin(field_type) in union_types
+
+
+def _extract_annotation(metadata: tuple) -> declarative_asn1.Annotation:
+    default = None
+    for raw_annotation in metadata:
+        if isinstance(raw_annotation, declarative_asn1.Default):
+            default = declarative_asn1.Default(value=raw_annotation.value)
```

Comment:
What's this actually doing? It looks like we already have an instance of the right type?